app/scripts/modules/amazon/src/serverGroup/details/AmazonServerGroupActions.tsx

```diff
@@ -140,23 +158,45 @@ export class AmazonServerGroupActions extends React.Component<IAmazonServerGroup
   };
 
   private rollbackServerGroup(): void {
-    const { app, serverGroup } = this.props;
+    const app = this.props.app;
```

Comment:
It's okay to use destructuring to get a single property out of another property (not necessary but totally fine)